docs/source/content/overview/linear-regression.rst

```diff
@@ -6,81 +6,83 @@ Linear Regression
   :local:
   :depth: 3
 
+*******
+Summary
+*******
+Linear regression is a technique used to analyze a linear relationship between input variables and a single output variable. We measure how accurate the resulting model is using a cost function. We want to minimize the value of the cost function in order to get a more accurate model.
+
 ********
 Overview
 ********
-**Linear regression** is a technique used to analyze a **linear relationship** between **input/independent/x** variables and a single **output/dependent/y** variable. A **linear relationship** means that the data points tend to follow a straight line. **Simple linear regression** involves only a single input variable.
-
-.. image:: _img/LR.png
+**Linear regression** is a technique used to analyze a **linear relationship** between **input** variables and a single **output** variable. A **linear relationship** means that the data points tend to follow a straight line. **Simple linear regression** involves only a single input variable.
 
-Our goal is to find the line that best models the path of the data points called a line of best fit [*Equation 1*].
+.. figure:: _img/LR.png
+   
+   *Figure 1*. A sample data set with a linear relationship
 
-*Equation 1*:
+Our goal is to find the line that best models the path of the data points called a line of best fit [*Equation 1, Figure 2*].
 
-.. image:: _img/Linear_Equation.png
+.. figure:: _img/Linear_Equation.png
+   
+   *Equation 1*. A linear equation
 
-.. image:: _img/LR_LOBF.png
+.. figure:: _img/LR_LOBF.png
+   
+   *Figure 2*. The dataset from *Figure 1* with a line of best fit
 
 Let’s break it down. We already know that x is the input value and y is our predicted output. a₀ and a₁ describe the shape of our line. a₀ is called the **bias** and a₁ is called a **weight**. Changing a₀ will move the line up or down on the plot and changing a₁ changes the slope of the line. Linear regression helps us pick appropriate values for a₀ and a₁.
 
-Note that we could have more than one input variable. In this case, we call it **multiple linear regression**. Let’s add another input variable called z.
-
-.. image:: _img/MLR.png
-
-Then the equation changes to a plane [*Equation 2*].
-
-*Equation 2*:
-
-.. image:: _img/3d_Linear_Equation.png
-
-.. image:: _img/MLR_POBF.png
-
-Adding extra input variables just means that we’ll need to find more weights. For this exercise, we will only consider a simple linear regression.
+Note that we could have more than one input variable. In this case, we call it **multiple linear regression**. Adding extra input variables just means that we’ll need to find more weights. For this exercise, we will only consider a simple linear regression.
```

Comment:
Could link out to a tutorial about multiple linear regression, this is up to you though. Something like:
```
If you're interested in learning about multiple linear regression, you can take a look at this guide:
https://towardsdatascience.com/super-simple-machine-learning-by-me-multiple-linear-regression-part-1-447800e8b624
```